src/cli/cli.cpp

```diff
@@ -1520,6 +1520,11 @@ void Interpreter::ProcessScan(int argc, char *argv[])
     AppendResult(error);
 }
 
+void Interpreter::s_HandleActiveScanResult(otActiveScanResult *aResult)
+{
+    Uart::sUartServer->mInterpreter.HandleActiveScanResult(aResult); // BUGBUG - Callbacks must provide a context
```

Comment:
Can you elaborate a little more on this comment? As I'm taking this all in it isn't immediately obvious to me what specifically the intended next action would be.